### ISSUE TITLE
Workflow editor (designer view) not showing

### ISSUE THREAD

**Issue by Georgeskh** [elsa-workflows/elsa-core#263]
I'm running Dashboard.web project from src code and the workflow design view is not showing.
![image](https://user-images.githubusercontent.com/49345041/76758861-3aa58180-6793-11ea-89c1-0e374e306040.png)

Any help on that matter would be greatly appreciated.

**Comment by sfmskywalker** [comment]
Make sure to also build the client-side assets as described [here](https://github.com/elsa-workflows/elsa-core#building--running-elsa-workflows-dashboard).

Let me know if that works or not.

**Comment by Georgeskh** [comment]
For step 2 I only did npm install for argon-dashboard in themes. I did not find the install.js script in root.

Other than that I followed the steps normally.

**Comment by sfmskywalker** [comment]
So you did `npm install` and then `gulp build`, but that didn't work? Do you at least see the assets created by gulp in the `wwwroot` folder of the `Elsa.Dashboard` folder?

If so, then try deleting all `bin` and `obj` folders and build the solution again. It sometimes happened to me that the contents of `wwwroot` of the Elsa.Dashboard` project would not be embedded in the binaries for some reason. Only after deleting the binaries and rebuilding did they become embedded.

**Comment by Georgeskh** [comment]
![image](https://user-images.githubusercontent.com/49345041/76761232-c02b3080-6797-11ea-8ef3-6cb7e97e8ee9.png)
As you can see it did not work.
I can see the assets here
![image](https://user-images.githubusercontent.com/49345041/76761304-de912c00-6797-11ea-87f3-78929622cc38.png)
and here 
![image](https://user-images.githubusercontent.com/49345041/76761344-efda3880-6797-11ea-8398-0fd2e99e15f0.png)

**Comment by sfmskywalker** [comment]
You're absolutely right - the asset building steps clearly succeeded, otherwise you wouldn't even see proper CSS applied.

Can you open F12 Dev tools and see if anything's up in the console?

**Comment by Georgeskh** [comment]
![image](https://user-images.githubusercontent.com/49345041/76762957-ed2d1280-679a-11ea-96de-0284a9988bca.png)
apparently it is not loading workflow designer

**Comment by sfmskywalker** [comment]
I can indeed reproduce. Will look into it.

**Comment by sfmskywalker** [comment]
I see what happened. I pushed an unstable 2.0.0-preview version of the updated designer to NPM, while the package.json in Elsa.Dashboard references "latest", effectively pulling down a broken version.

I just pushed a fix to `master`. When you pulled latest, perform these steps:

1. `npm install` (will download `"@elsa-workflows/elsa-workflow-designer": "0.0.61"`)
2. `gulp build`
3. Restart the ASP.NET Core project (Elsa.Dashboard.Web).

Then it works.
